cleanrl/ppo.py

```diff
@@ -101,14 +107,22 @@ class Agent(nn.Module):
     def __init__(self, envs):
         super().__init__()
         self.critic = nn.Sequential(
-            layer_init(nn.Linear(np.array(envs.single_observation_space.shape).prod(), 64)),
+            layer_init(
```

Comment:
you probably want to avoid all the reformatting in your PR, as it has nothing to do with your change